styleguide.css

```diff
@@ -323,29 +335,66 @@ form#signin {
 }
 
 [type=checkbox]:not(:checked) + label:after {
-    opacity   : 0;
-    transform : scale(0);
+    opacity: 0;
+    transform: scale(0);
 }
 
 [type=checkbox]:checked + label:after {
-    opacity   : 1;
-    transform : scale(1);
+    opacity: 1;
+    transform: scale(1);
 }
 
-
 [type=checkbox]:checked:focus + label:before,
-[type=checkbox]:not(:checked):focus + label:before
-{
-    border : 0.1rem solid #bde513;
+[type=checkbox]:not(:checked):focus + label:before {
+    border: 0.1rem solid #bde513;
 }
 
 label:hover:before {
     border : 0.1rem solid #bde513 !important;
     transition: border-color .3s;
 }
 
+textarea {
+    padding: 1.5rem;
+    border: 0.1rem solid #00283C;
+    min-width: 30rem;
+    min-height: 15rem;
+    max-width: 100%;
+    transition: border-color .3s;
+}
+
+.select-container{
```

Comment:
space before `{`.
However there may be other options than setting a wrapper for a select. This was for the arrow, therefore some trick like a simple just a negative left margin might do the job.